lib/inertia_rails/engine.rb

```diff
@@ -12,5 +12,20 @@ class Engine < ::Rails::Engine
         include ::InertiaRails::Controller
       end
     end
+
+    initializer "inertia_rails.subscribe_to_notifications" do
+      if Rails.env.development? || Rails.env.test?
```

Comment:
I want the default experience to add warnings to the logs in development / test _only_.